src/ConnectionAutomaton.cpp

```diff
@@ -320,13 +320,13 @@ void ConnectionAutomaton::onConnectionFrame(
   switch (type) {
     case FrameType::KEEPALIVE: {
       Frame_KEEPALIVE frame;
-      if (!deserializeFrameOrError(frame, std::move(payload))) {
+      if (!deserializeFrameOrError(isResumable_, frame, std::move(payload))) {
```

Comment:
Ah, I assumed that isResumable_ was a fixed value. Didn't realize it could change or differ from the other side. I'll fix.